policies/rudderc/templates/technique.ps1.askama

```diff
@@ -42,7 +46,7 @@ function {{ id }}
                 {{- arg.0 }} = "{{ arg.1 }}"
             {% endfor %}
         }
-        $call = {{ m.name }} @methodParams
+        $call = {{ m.name | dsc_case }} FIXME -PolicyMode $policyMode
```

Comment:
We can keep the '@methodParams' here instead of the '-PolicyMode $policymode'.

Reply:
not sure to understand, I'll let you review newer version of th ePR